function.js

```diff
@@ -0,0 +1,128 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let className = document.querySelector("#className")
+    let teacher = document.querySelector("#teacherName")
+    let stClass = document.querySelector("#class")
+    let StName = document.querySelector("#name")
+    let age = document.querySelector("#age")
+    let city = document.querySelector("#city")
+    let grade = document.querySelector("#grade")
+    let btn = document.querySelector("#show")
+
+
+    const showEverything = async () =>{
+        try {
+            let show = await axios.get("http://localhost:3000/").then((res)=>{
```

Comment:
The point of using async and await is to eliminate the need for using `.then` after a promise.

In this case the `res` in the `.then` would be the exact value as your variable `show`

If done without using .then it would look like this

`let show = await axios.get("http://localhost:3000/")`
` let p = document.createElement("p")`
`p.innerText = show.data`